src/declarativetabmodel.cpp

```diff
@@ -378,6 +378,22 @@ void DeclarativeTabModel::updateActiveTab(const Tab &activeTab, bool loadActiveT
     }
 }
 
+void DeclarativeTabModel::setWebContainer(DeclarativeWebContainer *webContainer)
```

Comment:
Those must be still registered but not as creatable types. Will add a commit on top. Above method is only used from the unit tests at the moment.